### PR TITLE
feat: enable multicolor strokes

### PR DESCRIPTION
**Pull request by TsubasaK111**
- helps express danger / passways more intuitive for sketches

### CODE REVIEW

File: js/components/Draw.js [hunk 1]
```diff
@@ -24,34 +27,62 @@ export default class Draw extends Component {
...
+            reverse
+            color='#F9A800'
+            size={18}
+            onPress={this._changeColor('#F9A800')}
+          />
+          <Icon
+            reverse
+            color='#9B2423'
+            size={18}
+            onPress={this._changeColor('#9B2423')}
```
Comment (takayukioda) [inline js/components/Draw.js:63]: I think it's better to give some name to those colors.

That'll prevent having different color on button and real color as well.

**Review by takayukioda — COMMENTED** [review]
Small improvement suggestion :P

File: js/components/Draw.js (hunk 1 above)
Comment (TsubasaK111) [reply js/components/Draw.js:63]: true that!

**Review by DennoDin — APPROVED** [review]
Functionally looks good to me. 
The Red and Green colors you picked are not colorblind friendly.
Neither I nor Dylan could tell the difference between them.
Will request a change to colors in a user issue

**Comment by DennoDin** [comment]
Please change colors for colorblind visibility.

**Pull request merged**